src/fidesops/api/v1/endpoints/user_endpoints.py

```diff
@@ -100,6 +100,9 @@ def user_login(
             status_code=HTTP_403_FORBIDDEN, detail="Incorrect password."
         )
 
+    # perform what's needed on the user model object
+    user.login(db)
+
     client: ClientDetail = user.client
```

Comment:
It feels strange to me to have this login method that only sets the `last_login_at` field but doesn't actually do any of the "logging in", which happens by creating this separate ClientDetail resource below.  There are a few ways to organize this (because the logging in involves updating two models), but my thought is that having all the "logging in" logic in one method is more understandable.

Reply:
yeah, that makes a lot of sense. i should have taken a more comprehensive look at the whole workflow here. your suggestion seems good to me - let me get a more holistic understanding of the user/client framework, and i'll adjust accordingly.

Reply:
addressed in https://github.com/ethyca/fidesops/pull/394/commits/308a53035a3edffbab869a5649a1c12a4ea5ebfd 

note that our our first choice had been to move the `ClientDetail` instantiation into the `login` method on the `FidesopsUser` class, but doing so introduced a circular dependency between `ClientDetail` and `FidesopsUser` -- the fk specification on `ClientDetail` refers to the `FidesopsUser.id_field_path` class field. so @pattisdr and i decided on the solution in the above commit.